README.rst

```diff
@@ -23,12 +23,7 @@ It can be installed with::
 
 Useful tips:
 
-* Be sure to install it for a version of Python which is compatible
-  with your codebase: for Python 2, ``pip2 install pyflakes`` and for
-  Python3, ``pip3 install pyflakes``.
```

Comment:
this first part is still important but should be replaced with `python#.# -m pip ...`